src/components/DataTable/examples/07.empty-with-custom-title-and-body.jsx

```diff
@@ -15,7 +15,7 @@ export default createClass({
 		const { data } = this.state;
 
 		return (
-			<DataTable data={data} density="extended" isFullWidth>
+			<DataTable data={data} density="extended" isFullWidth minRows={15}>
```

Comment:
That one has an incomplete page of data, so you can see minRows fill in the rest
The bug in this case was when data = [], it wouldn't fill out the minRows count, but just did a hardcoded 10 rows
so this covers that case specifically where you have no data and also minRows > 10